src/app/containers/CpsFeaturesAnalysis/index.jsx

```diff
@@ -23,20 +23,41 @@ const PromoListComponent = ({ promoItems, dir }) => {
   const viewRef = useViewTracker(eventTrackingData.block);
   const { isAmp } = useContext(RequestContext);
 
-  const StoryPromo = isAmp || isLive() ? _StoryPromo : FrostedGlassPromo;
+  const promoVariation = useOptimizelyVariation(
+    'high_impact_feature_analysis_promo',
+  );
+
+  let hasFrostedGlassPromo = false;
+
+  if (promoVariation) {
+    if (promoVariation === 'variation_1' && !isAmp) {
+      hasFrostedGlassPromo = true;
+    }
+  }
 
   return (
     <StoryPromoUl>
-      {promoItems.map(item => (
+      {promoItems.map((item, i) => (
```

Comment:
Could remove the prop duplication here using a pattern like:

```javascript
{promoItems.map((item, i) => {
    const PromoComponent = i === 0 && hasFrostedGlassPromo ? FrostedGlassPromo : StoryPromo;
    return <PromoComponent
              item={item}
              dir={dir}
              displayImage
              displaySummary={false}
              serviceDatetimeLocale={serviceDatetimeLocale}
              eventTrackingData={eventTrackingData}
            />
}
```

Reply:
can we rename i to index?

Reply:
Done! Tested it locally and all seems to be fine. Wrapped the `PromoComponent` in `StoryPromoLi `as well